nrf-sdc/src/sdc.rs

```diff
@@ -1450,6 +1424,17 @@ pub mod vendor {
         }
     }
 
+    cmd! {
+        NordicSetEventStartTask(VENDOR_SPECIFIC, 0x11b) {
```

Comment:
That should be `0x011e`. As far as I understand there's some math involved.

Original opcode `id` is `0xfd1e`:
https://github.com/nrfconnect/sdk-nrfxlib/blob/main/softdevice_controller/include/sdc_hci_vs.h#L109

Reply:
Ah, I see. Thanks!

Reply:
And that hidden math is following: `VENDOR_SPECIFIC` marks Opcode group (`0x3f`). To end up with eventual opcode, we'll need to do following math: `opcode_group << 10 + opcode`, so `0x3f << 10 == 0xfc00`.